migrations/20231102101356_user.py

```diff
@@ -23,14 +57,64 @@ def upgrade(db: "pymongo.database.Database"):
                 "_id": user['_id'],
                 "email": user['profile']['email'],
                 "hashed_password": user['profile']['hashed_password'],
-                "is_active": True,
-                "is_superuser": False,
-                "is_verified": False,
+                "is_active": user['active'],
+                "is_superuser": is_superuser,
+                "is_verified": is_superuser,
                 "username": user['profile']['username'],
                 "groups": user['profile']['groups']
             },
         )
+    # Sanity check: check if there are any old-format users in the
+    # "admin" group. Remove the group if there aren't any
+    remaining_admins = db.user.count(
```

Comment:
I don't think there will be any remaining admin users that are still in `admin` group after migration.
Do we have any specific use-case for this?